submission/programs/adadas.r

```diff
@@ -122,8 +101,61 @@ adas3 <- derive_vars_merged(
   )
 
 
-## baseline information BASE/CHG/PCHG
+## ANL01FL
 adas4 <- adas3 %>%
+  mutate(diff = AWTARGET - ADY) %>%
+  restrict_derivation(
+    derivation = derive_var_extreme_flag,
+    args = params(
+      by_vars = exprs(USUBJID, PARAMCD, AVISIT),
+      order = exprs(AWTDIFF, diff),
+      new_var = ANL01FL,
+      mode = "first"
+    ),
+    filter = !is.na(AVISIT)
+  )
+
+################# Test: derive LOCF only considering ANL01FL=Y
```

Comment:
Could we do a comment on why we this derivation is the way it is - feedback from FDA on need to match Pilot 1 - and maybe even provide github wiki qc finding link?

Reply:
@bms63 good point! updated